### PR TITLE
appbank.net

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

https://www.appbank.net/
 https://www.appbank.net/2022/09/09/iphone-application/2293666.php

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SocialFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -15,6 +15,7 @@ karapaia.com#?#.article-body > div[style^="text-align:center;"]:has(> iframe[src
 pokemongo-get.com#?#.post_bottom > div + div:has(> div.pg_board_post_button > i.fa.mr-5)
 psyche.co#?#div[class^="ideas__SocialShareBarWrapper-"] > div > p:has(> svg[data-cy$="-outline"])
 manga1001.club#?#.section.box:has(> .social_share_wrapper)
+appbank.net#?#.lets-push:contains(シェア)
```
Comment (Yuki2718) [inline SocialFilter/sections/css_extended.txt:18]: I guess simple `##.lets-push` to specific.txt is enough. Sorry if you know false positive by that and avoided.

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -2390,6 +2390,7 @@ pref.aomori.lg.jp##.pr_banner
 pref.niigata.lg.jp###top_banner_koukoku
 pref.aichi.jp##.top_kokoku_box
 appbank.net##.appbank-tags > a.RWh2:first-child
+appbank.net##.entry-list-item-native-ad
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:2393]: Add also `##.body-ad` seen in the same page as `.lets-push`.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/css_extended.txt (hunk 1 above)
Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt:18]: https://github.com/AdguardTeam/AdguardFilters/pull/129381#discussion_r967863484
The reason is the same as this.
Am I worrying about it too much?

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2393]: I see. Done

File: SocialFilter/sections/css_extended.txt (hunk 1 above)
Comment (Yuki2718) [reply SocialFilter/sections/css_extended.txt:18]: If `appbank.net##.lets-push` is safe on all `appbank.net` pages, it'll be preferred.

Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt:18]: I will change rule to  `appbank.net##.lets-push`

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**